src/main/java/org/jenkinsci/plugins/workflow/pipelinegraphanalysis/StatusAndTiming.java

```diff
@@ -428,10 +431,16 @@ private int parseIota(FlowNode node) {
         System.out.println("Action format: ");
         System.out.println("\t- actionClassName actionDisplayName");
         System.out.println("------------------------------------------------------------------------------------------");
+        Function<FlowNode, String> flowNodeToId = new Function<FlowNode, String>(){
+            @Override
+            public String apply(@Nullable FlowNode input) {
+                return (input != null) ? input.getId() : null;
+            }
+        };
         for (FlowNode node : sorted) {
             StringBuilder formatted = new StringBuilder();
             formatted.append('[').append(node.getId()).append(']');
-            formatted.append('{').append(StringUtils.join(node.getParentIds(), ',')).append('}');
+            formatted.append('{').append(StringUtils.join(Collections2.transform(node.getParents(), flowNodeToId), ',')).append('}');
```

Comment:
What is the purpose of this change? Seems to be a more complicated way of writing the same thing.